products/compute/api.yaml

```diff
@@ -1057,22 +1057,6 @@ objects:
           characters must be a dash, lowercase letter, or digit, except the last
           character, which cannot be a dash.
         required: true
-      - !ruby/object:Api::Type::NameValues
```

Comment:
Docs say that this resource does not have labels or set_labels endpoint.

Reply:
Drive-by but this is GlobalAddress, right? It's present at beta if so.

https://cloud.google.com/compute/docs/reference/rest/beta/globalAddresses/setLabels